README.md

```diff
@@ -53,6 +53,38 @@ You can first localize the robot by initializing the pose and then setting a nav
 
 [![](media/localize.png)](media/localize.mp4)
 
+## Running the reconnaissance demos:
+
+### Host/container insider attacker
+``` bash
+rocker rosswg/turtlebot3_demo "byobu -f configs/unsecure.conf attach"
+```
+
+### Internal network attacker
+``` bash
+# in Terminal (terminal 1), initialize first a swarm
+docker swarm init
+# in Terminal (terminal 1), create the network overlay
+docker network create -d overlay \
+  --subnet=10.0.0.0/24 \
+  --gateway=10.0.0.1 \
+	--ip-range 10.0.0.192/27 \
+  --attachable \
+  overlay
+
+# in another Terminal (terminal 2), launch demo
+rocker --network overlay rosswg/turtlebot3_demo "byobu -f configs/unsecure.conf attach"
+
+# in another Terminal (terminal 3), launch another container
+docker run -it --rm --network overlay --name aztarna rosswg/turtlebot3_demo /bin/bash
```

Comment:
What does the user need to do/type after launching bash?

Reply:
I plan to walk audience through it but I've added a simple scan for completeness. 1dc426384a8c20270d885c01a4da495e3763c213.